main/helper.py

```diff
@@ -0,0 +1,39 @@
+
+from .models import *
+
+
+def get_invalid_matches():
+    matches = Match.objects.all()
+    invalid_matches = []
+    for match in matches:
+        if(match.player1_score != 11 and match.player2_score != 11):
+            invalid_matches.append(match)
+            continue
+
+        if(match.player1_score > 11 or match.player2_score > 11):
+            invalid_matches.append(match)
+            continue
+
+        if(match.player1_score < 0 or match.player2_score < 0):
+            invalid_matches.append(match)
+            continue
+
+    return(invalid_matches)
+
+
+def remove_invalid_matches():
+    invalid_matches = get_invalid_matches()
+    for match in invalid_matches:
+        match.delete()
+
+
+def get_match_details(match):
+    return {
+        "id": match.id,
+        "date_and_time": match.date_and_time,
+        "player1": match.player1.name,
+        "player2": match.player2.name,
+        "player1_score": match.player1_score,
+        "player2_score": match.player2_score,
+        "field": match.field.name
+    }
```

Comment:
Please rename this file to `helpers.py` (instead of `helper.py`) because that's the convention. Keep in mind that you're going to have to update all your import statements as well, so I'd definitely recommend not doing this by hand, but using IntelliJ, which can automatically do all of that for you (right click on the file, Refactor, Rename I believe).